src/test/java/marquez/service/DatasetServiceTest.java

```diff
@@ -1,3 +1,157 @@
 package marquez.service;
```

Comment:
Please add license info.

Reply:
We can use `spotless` to help us catch missing license headers. Something worth looking into.